.circleci/config.yml

```diff
@@ -45,6 +48,33 @@ workflows:
               ignore: /.*/
 
 jobs:
+  package_crypto:
+    docker:
+      - image: rust:1.47.0
```

Comment:
Could you rebased this on current master? We use 1.49.0 now as the min Rust version.